packages/core/src/components/MenuToggle/index.tsx

```diff
@@ -161,29 +164,43 @@ export class MenuToggle extends React.Component<Props & WithStylesProps, State>
 
     const Button = muted ? MutedButton : BaseButton;
 
+    let toggleButton = toggleIcon ? (
+      <IconButton
+        disabled={disabled}
+        aria-label={accessibilityLabel}
+        onClick={this.handleToggleMenu}
+      >
+        {toggleIcon}
+      </IconButton>
+    ) : (
+      <Button
+        disabled={disabled}
+        afterIcon={<ExpandableIcon expanded={opened} size={iconSize} />}
+        inverted={inverted}
+        large={large}
+        small={small}
+        onClick={this.handleToggleMenu}
+      >
+        {toggleLabel}
+      </Button>
+    );
+
+    if (toggleElement) {
+      toggleButton = (
+        <span
```

Comment:
or make this a `button` and reset all the styles with `initial` or `inherit` or `none` or `0` padding/margins where appropriate? really not a fan of clicks on spans/divs

Reply:
Got it